lib/mongo/error/no_server_available.rb

```diff
@@ -27,9 +27,11 @@ class NoServerAvailable < Error
       #
       # @param [ Hash ] server_selector The server preference that could not be
       #   satisfied.
+      # @param [ Cluster ] cluster The cluster that server selection was
+      #   performed on. (added in 2.7.0)
       #
       # @since 2.0.0
-      def initialize(server_selector)
+      def initialize(server_selector, cluster=nil)
```

Comment:
Ditto question on #1011; why is the cluster parameter here?

Reply:
Must've gotten killed off.